test.sh

```diff
@@ -174,12 +174,20 @@ if [[ "$RUN" =~ "vet" ]] ; then
   end_context #vet
 fi
 
+function run_golint() {
+  autogen_regexp="(mockstatsd)"
```

Comment:
The idea is we update this each time we add a new mock? Seems fragile.

For previous mock code I had it output as foo_test.go, which I think made lint ignore it.